test/helpers/position_assertions.cc

```diff
@@ -40,7 +40,7 @@ const UnorderedMap<
         {"enable-packager", BooleanPropertyAssertion::make},
         {"enable-experimental-requires-ancestor", BooleanPropertyAssertion::make},
         {"enable-suggest-unsafe", BooleanPropertyAssertion::make},
-        {"exhaustive-apply-code-action", BooleanPropertyAssertion::make},
+        {"selective-apply-code-action", SelectiveApplyCodeActionAssertions::make},
```

Comment:
Technically, I guess this should be `SelectiveApplyCodeActionPropertyAssertion`, in trying to keep with `*PropertyAssertion` as sigils/directives at the beginning of the file?  (I don't know why they're called "PropertyAssertions", because they're not asserting anything, but that is something to fixup for another day.

Correspondingly, I guess `apply-code-action` should really become `apply-quickfix`, for consistency with the changes made in this PR?  (And `ApplyCodeActionAssertion` should become `ApplyQuickfixAssertion`?)

Reply:
>Correspondingly, I guess apply-code-action should really become apply-quickfix, for consistency with the changes made in this PR? (And ApplyCodeActionAssertion should become ApplyQuickfixAssertion?)

I don't think it would mirror the LSP behaviour correctly. Quickfixes are a subset of code actions. For example, the `refactor.extract` action is a code action, but not a quickfix, and it still uses the `ApplyCodeActionAssertion `.